php/Dockerfile-7.0

```diff
@@ -26,6 +26,7 @@ RUN apt-get update -q -y \
         libmemcachedutil2 \
 # Required by eZ Commerce
         imagemagick \
+        libxml2-dev \
```

Comment:
build deps we usually put in `buildDeps`
Is there some more specific package you need after compilation + this one in buildDeps perhaps?

Reply:
Thanks. Indeed the image becomes much smaller with this approach.
Corrected.